internal/backends/winit/lib.rs

```diff
@@ -369,61 +368,86 @@ pub(crate) struct SharedBackendData {
     #[cfg(enable_skia_renderer)]
     skia_context: i_slint_renderer_skia::SkiaSharedContext,
     active_windows: RefCell<HashMap<winit::window::WindowId, Weak<WinitWindowAdapter>>>,
+    inactive_windows: RefCell<Vec<Weak<WinitWindowAdapter>>>,
```

Comment:
```suggestion
    /// List of visible windows that have been created when without the event loop and 
    /// need to be mapped to a winit Window as soon as the event loop becomes active.
    inactive_windows: RefCell<Vec<Weak<WinitWindowAdapter>>>,
```

If i understood correctly.